python/ql/lib/semmle/python/dataflow/new/internal/TypeTracker.qll

```diff
@@ -224,6 +224,50 @@ private module Cached {
 
 private import Cached
 
+private predicate step(TypeTrackingNode nodeFrom, TypeTrackingNode nodeTo, StepSummary summary) {
+  stepNoCall(nodeFrom, nodeTo, summary)
+  or
+  stepCall(nodeFrom, nodeTo, summary)
+}
+
+pragma[nomagic]
+private predicate stepProj(TypeTrackingNode nodeFrom, StepSummary summary) {
+  step(nodeFrom, _, summary)
+}
+
+bindingset[nodeFrom, t]
+pragma[inline_late]
+pragma[noopt]
```

Comment:
"Jamen, det er jo hele tre ting på én gang! Det går da virkelig ikke."

(With apologies to those in the audience who did not grow up with Danish commercials for Kinder eggs.)